src/test/scala/tech/cryptonomic/conseil/routes/PlatformDiscoveryTest.scala

```diff
@@ -148,20 +148,31 @@ class PlatformDiscoveryTest extends WordSpec with Matchers with ScalatestRouteTe
         PlatformConfiguration(None, Some(true), None, Map("mainnet" ->
           NetworkConfiguration(None, Some(true), None, Map("entity" ->
             EntityConfiguration(None, Some(true), None, Map("attribute" ->
-              AttributeConfiguration(None, Some(true), Some("description"), Some("placeholder"), Some("dataFormat")))))))))
+              AttributeConfiguration(
+                displayName = None,
+                visible = Some(true),
+                description = Some("description"),
+                placeholder = Some("placeholder"),
+                scale = Some(6),
+                dataType = Some("hash"),
+                dataFormat = Some("dataFormat"),
+                valueMap = Some(Map("0" -> "value"))))))))))
 
       // when
       Get("/v2/metadata/tezos/mainnet/entity/attributes") ~> addHeader("apiKey", "hooman") ~> sut(overridesConfiguration) ~> check {
 
         // then
         status shouldEqual StatusCodes.OK
         contentType shouldBe ContentTypes.`application/json`
-        val result: List[Map[String, String]] = toListOfMaps[String](responseAs[String])
+        val result: List[Map[String, Any]] = toListOfMaps[Any](responseAs[String])
         result.head("name") shouldBe "attribute"
```

Comment:
Trivial: just realized this can get confusing for some: maybe store the head once and reuse it?
```scala
val headResult: Map[String, Any]= toListOfMaps[Any](responseAs[String]).head
headResult("name")...
...
```